src/systems/ec/implementations/vanilla/packages/ec-component-site-header-core/ec-component-site-header-core.scss

```diff
@@ -78,6 +78,12 @@
     }
   }
 
+  .ecl-site-header-core__login-toggle {
```

Comment:
I'd personally suggest this to be `ecl-site-header-core__login-toggle-arrow` or `ecl-site-header-core__login-arrow`, there are many `icon`-s already. Just a suggestion, not a blocker.

Reply:
Btw, the code after `{logged && ()}` is a candidate for a separate component by itself, as `logged` prop is wrapping quite some logic already.

Reply:
@kalinchernev thanks for the suggestion, updated to `ecl-site-header-core__login-arrow`